backend/common/ai_drive.py

```diff
@@ -210,31 +209,40 @@ def ml_drive(
         problem_type (str): "classification" or "regression" problem
```

Comment:
should update docs to reflect the trainspace_data params for these `ai_drive` functions